pkg/kubectl/service_basic_test.go

```diff
@@ -152,3 +153,174 @@ func TestServiceBasicGenerate(t *testing.T) {
 		}
 	}
 }
+
+func TestParsePorts(t *testing.T) {
+	tests := []struct {
+		portstring       string
+		expectport       int32
+		expecttargetport intstr.IntOrString
+		expecterr        string
+	}{
+		{
+			portstring:       "3232",
+			expectport:       3232,
+			expecttargetport: intstr.IntOrString{Type: intstr.Int, IntVal: 3232},
+			expecterr:        "",
```

Comment:
Perhaps, you can omit it at all (`expecterr` will be `nil` in that case and should work the same as an empty string).

Reply:
@php-coder Done ,Thanks for reviewing